PC/msvcrtmodule.c

```diff
@@ -415,7 +415,7 @@ msvcrt_ungetwch_impl(PyObject *module, int unicode_char)
 
 #endif /* MS_WINDOWS_DESKTOP */
 
-#ifdef _DEBUG
+#ifdef Py_DEBUG
```

Comment:
I'd leave this one alone. The presence of `CrtSetReportFile` is directly based on `_DEBUG`, rather than our own variable.